core/src/main/java/org/apache/sedona/core/joinJudgement/DynamicIndexLookupJudgement.java

```diff
@@ -71,7 +59,7 @@ public DynamicIndexLookupJudgement(
             Metric resultCount,
             Metric candidateCount)
     {
-        super(spatialPredicate);
+        super(spatialPredicate, buildCount, streamCount, resultCount, candidateCount);
```

Comment:
The metric members `buildCount`, `streamCount`, etc. has been moved to the base class. We can remove these metrics members from `DynamicIndexLookupJudgement`.